app/move/steps/create.js

```diff
@@ -178,6 +178,30 @@ module.exports = {
           'move-date',
         ],
       },
+      {
+        field: 'from_location_type',
+        value: 'secure_childrens_home',
+        next: [
+          {
+            field: 'to_location_type',
+            value: 'secure_childrens_home',
+            next: 'move-date-range',
+          },
+          'move-date',
```

Comment:
Are all these `move_date`s necessary? (I know that the original one here is for the prison type and that it's been cloned). Surely the final`move_date` value is sufficient?

Reply:
> Are all these `move_date`s necessary? (I know that the original one here is for the prison type and that it's been cloned). Surely the final`move_date` value is sufficient?

Nope. I'm completely wrong - that is really unnecessarily verbose :(